packages/core/package.json

```diff
@@ -205,8 +205,8 @@
     "@keystone-ui/tooltip": "workspace:^",
     "@nodelib/fs.walk": "^2.0.0",
     "@prisma/client": "catalog:",
-    "@prisma/internals": "5.17.0",
-    "@prisma/migrate": "5.17.0",
```

Comment:
This is a follow-up to https://github.com/keystonejs/keystone/pull/9215, which missed some dependencies